src/main/java/org/junit/Assert.java

```diff
@@ -603,23 +603,17 @@ public static void assertNotEquals(String message, float unexpected,
     private static boolean doubleIsDifferent(double d1, double d2, double delta) {
         if (Double.compare(d1, d2) == 0) {
             return false;
+        }else{
```

Comment:
This is slightly better. Personally I wouldn't use "else" here (unnecessary line leading to unnecessary indentation.

Note that in Java generally keywords are surrounded by a single space (we considered having auto-formatting or lint warnings for this, but that would introduce a lot of churn)